bin/pycbc_live

```diff
@@ -77,16 +77,50 @@ class LiveEventManager(object):
         else:
             raise TypeError("Not root process")
 
+    def compute_followup_data(self, ifos, coinc_results, data_readers, bank,
+                              followup_ifos=None):
+        """Figure out which of the followup detectors are usable, and compute
+        SNR time series for all the available detectors.
+        """
+        out = {}
+
+        template_id = coinc_results['foreground/{}/template_id'.format(ifos[0])]
+        htilde = bank[template_id]
+
+        subthreshold_sngl_time = numpy.mean(
+                [coinc_results['foreground/{}/end_time'.format(ifo)]
+                 for ifo in ifos])
+
+        for ifo in ifos + (followup_ifos or []):
+            if ifo in ifos:
+                trig_time = coinc_results['foreground/{}/end_time'.format(ifo)]
+            else:
+                trig_time = subthreshold_sngl_time
+            # NOTE we only check the state/DQ of followup IFOs here.
+            # IFOs producing the coincidence are assumed to also
+            # produce valid SNR series.
+            snr_series, psd = compute_followup_snr_series(
+                    data_readers[ifo], htilde, trig_time,
+                    check_state=(ifo in followup_ifos))
+            if snr_series is not None:
+                out[ifo] = {'snr_series': snr_series, 'psd': psd}
+
+        return out
+
     def check_coincs(self, ifos, coinc_results, psds, low_frequency_cutoff,
                      data_readers, bank, followup_ifos=None):
         """ Save zerolag triggers to a coinc xml file """
         if 'foreground/ifar' in coinc_results: 
             ifar = coinc_results['foreground/ifar']
-            event = SingleCoincForGraceDB(
-                    ifos, coinc_results, data_readers=data_readers,
-                    bank=bank, upload_snr_series=args.upload_snr_series,
-                    followup_ifos=followup_ifos,
-                    gracedb_server=args.gracedb_server)
+
+            event_kwa = {'bank': bank,
+                         'gracedb_server': args.gracedb_server}
+
+            if args.upload_snr_series:
```

Comment:
Should we still allow it to be optional to upload the snr time series? Perhaps we should just require it and drop all the extra logic?

Reply:
I am trying to imagine a weird problem with a future, loud, multidetector event which forces us to disable the SNR time series and just use the trigger point estimates (we know how nature is). In that case however I think it should be BAYESTAR's responsibility to have the option to ignore the SNR series. So yes, I will just make it permanent from the PyCBC side.